internal/api/v1/router.go

```diff
@@ -45,6 +45,12 @@ const (
 // The "Status" of the first error in the list becomes the response Status Code.
 type APIActionFunc func(c *gin.Context) errors.APIErrors
 
+// ErrorHandler exposes the internal errorhandler for use by the server without having to edit the
+// router definitions.
+func ErrorHandler(action APIActionFunc) gin.HandlerFunc {
+	return errorHandler(action)
+}
+
```

Comment:
I see that this was probably done to avoid changing all the refeference to the `errorHandler`. We can change that if you want to. Otherwise that's fine to me.